devolutions-gateway/src/recording/mod.rs

```diff
@@ -614,6 +631,13 @@ async fn recording_manager_task(
                             id,
                         });
 
+                        if RecordingFileType::WebM == file_type {
+                            tokio::spawn(async move {
+                                if let Err(e) = remux(recording_file).await {
+                                    error!(error = format!("{e:#}"), "remux");
+                                }
+                            });
+                        }
```

Comment:
suggestion: Move that into `handle_disconnect`, and use the `current_file.file_name` instead.

```rust
            let recording_file_path = ongoing
                .manifest_path
                .parent()
                .expect("a parent")
                .join(&current_file.file_name);

            debug!(path = %ongoing.manifest_path, "Write updated manifest to disk");

            ongoing
                .manifest
                .save_to_file(&ongoing.manifest_path)
                .with_context(|| format!("write manifest at {}", ongoing.manifest_path))?;

            if recording_file_path.extension() == Some(RecordingFileType::WebM.extension()) {
                tokio::spawn(async move {
                    if let Err(e) = remux(recording_file_path).await {
                        error!(error = format!("{e:#}"), "Remux operation failed");
                    }
                });
            }

```